helm/aws-operator-chart/templates/03-configmap.yaml

```diff
@@ -29,6 +29,22 @@ data:
           enabled: '{{ .Values.Installation.V1.Provider.AWS.TrustedAdvisor.Enabled }}'
         vaultAddress: '{{ .Values.Installation.V1.Auth.Vault.Address }}'
         vpcPeerID: '{{ .Values.Installation.V1.Provider.AWS.VPCPeerID }}'
+      cluster:
+        calico:
+          cidr: '{{ .Values.Installation.V1.Guest.Calico.CIDR }}'
+          mtu: 1430
```

Comment:
I somehow agree with you. I would just like to keep it like that because there are more values like this already and the scheme/direction is not really clear to me. I would then also like to have a more structured approach in `values.yaml` but that would mean more unrelated changes. I copied this from `kubernetesd` just how it is.